apps/backend/.env.example

```
@@ -22,12 +22,13 @@ TOKEN_ALGORITHM=HS256
 TOKEN_EXPIRES_IN=24h
 
 #
-# AWS S3
+# AWS
 #
 AWS_ACCESS_KEY_ID=aws_access_key_id
 AWS_SECRET_ACCESS_KEY=aws_secret_access_key
 S3_REGION=s3_region
```

```suggestion
AWS_REGION=s3_region
```
you now use it more that just for s3